<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonC">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDiv">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultp">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSub">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>220</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSum">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDot">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton0">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRes">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRoot">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>âˆš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonLBrack">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>(</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRBrack">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>385</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="themeDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="themeLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
